griptape/drivers/vector/redis_vector_store_driver.py

```diff
@@ -129,14 +130,14 @@ def query(
 
         query_params = {"vector": np.array(vector, dtype=np.float32).tobytes()}
 
-        results = self.client.ft(self.index).search(query_expression, query_params).docs
+        results = self.client.ft(self.index).search(query_expression, query_params).docs  # pyright: ignore
```

Comment:
Why the pyright ignore?

Reply:
`Argument of type "dict[str, bytes]" cannot be assigned to parameter "query_params" of type "_QueryParams | None" in function "search"` was the error, ignored hoping for some insight here